app/controllers/static_pages_controller.rb

```diff
@@ -5,7 +5,7 @@ def accessibility_statement
   def contact_us
   end
 
-  def cookies
+  def cookies_page
```

Comment:
this is so that calling `cookies` from the view calls the rails implementation rather than this controller action method